### ISSUE TITLE
🔤 String "Screening questions" in English is translated into French four different ways

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#8521]
## 🔤 Copy

The string "Screening questions" in English is translated into French four different ways.

1. `sTij/C` = "Question de sélection" (definitely, should be plural)
2. `qGyD4w` = "Questions de sélection"
3. `ccyVDQ` = "Questions de sélection"
4. `mqWvWR` = "Questions relatives à la sélection préliminaire "
5. `c+QwbR` = "Questions filtres"

> [!NOTE]  
> This list doesn't even take into account strings where the words **screening questions** is part of a larger sentence. 

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the copy has been added or updated appropriately.

- [ ] "Screening questions" in English translated as "questions de sélection"

**Comment by brindasasi** [comment]
wow.. thanks for finding this @mnigh 
I'm wondering how do we avoid this happening further? 
Should we maintain a approved french glossary for most repeated terms like this ?

**Comment by mnigh** [comment]
> wow.. thanks for finding this @mnigh I'm wondering how do we avoid this happening further? Should we maintain a approved french glossary for most repeated terms like this ?

an approved glossary in english and french would be great, but deciding who approves the copy is the trickiest part.

**Comment by RM-1978** [comment]
> > wow.. thanks for finding this @mnigh I'm wondering how do we avoid this happening further? Should we maintain a approved french glossary for most repeated terms like this ?
> 
> an approved glossary in english and french would be great, but deciding who approves the copy is the trickiest part.

I agree 100% @brindasasi.  We have asked the Editing and Translation team to work with us to create a French Glossary with the terms that are found in the Final french texts approved by Lauren.

**Comment by gobyrne** [comment]
@RM-1978 if you come to an agreement with the editors, let's implement.

**Comment by mnigh** [comment]
termium lists the following options for **screening question**:

- _question de sélection_ 
- _question filtre_ 

REF: https://www.btb.termiumplus.gc.ca/tpv2alpha/alpha-eng.html?lang=eng&i=1&srchtxt=SCREENING+QUESTION&codom2nd_wet=1#resultrecs

**Comment by RM-1978** [comment]
Please note that the French Translation chosen by the editors and Francophone subject matter professionals in the recruitment team is : `questions de sélection`
